### PR TITLE
feat(fa): File download

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...

https://app.asana.com/0/1200897217304888/1200385618296210

## What

Allows downloading of files in veita.
Makes filelist not have hover state nor clickable in Ósk.

## Why
Needed for handling of applications

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/financial-aid/shared/src/components/FileList/FileList.tsx [hunk 1]
```diff
@@ -9,6 +8,17 @@ import {
...
+import { gql, useLazyQuery } from '@apollo/client'
+
+export const GetSignedUrlQuery = gql`
+  query GetSignedUrlQuery($input: GetSignedUrlForIdInput!) {
+    getSignedUrlForId(input: $input) {
+      url
+      key
+    }
+  }
+`
```
Comment (Rikkiabb) [inline libs/financial-aid/shared/src/components/FileList/FileList.tsx:20]: Know this should be in sharedGql, but if this is there then we have a circular dependency 😭  We could add a sharedGql file into our shared lib, but this is the only on there so putting it on hold for now 🧑‍🤝‍🧑

File: libs/financial-aid/shared/src/components/FileList/FileList.tsx [hunk 2]
```diff
@@ -18,18 +28,35 @@ const FileList = ({ className, files }: Props) => {
...
+    },
+  })
+
   return (
     <Box className={cn({ [`${className}`]: true })} marginBottom={2}>
       <>
         {files.map((item, index) => {
           return (
-            <a
+            <Box
```
Comment (MargretFinnboga) [inline libs/financial-aid/shared/src/components/FileList/FileList.tsx:48]: div with an onclick event, isnt that bad practice? why not have it a tag ? Just curious👀

File: apps/financial-aid/backend/src/environments/environment.ts [hunk 3]
```diff
@@ -40,7 +41,8 @@ const devConfig = {
   files: {
     cloudFrontPublicKeyId: process.env.CLOUDFRONT_PUBLIC_KEY_ID ?? '',
     cloudFrontPrivateKey: process.env.CLOUDFRONT_PRIVATE_KEY ?? '',
-    fileBaseUrl: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
+    fileBaseUrlOsk: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
+    fileBaseUrlVeita: 'https://fjarhagsadstod.dev.sveitarfelog.net/files',
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/environments/environment.ts:45]: Should this be the same links? Then why have them twice? 👯‍♀️

Comment (Rikkiabb) [reply apps/financial-aid/backend/src/environments/environment.ts:45]: This a change that was here when I took over the PR, but great question, will change, this doesn't make sense

File: libs/financial-aid/shared/src/components/FileList/FileList.tsx (hunk 2 above)
Comment (Rikkiabb) [reply libs/financial-aid/shared/src/components/FileList/FileList.tsx:48]: True 💯 will change

**Review by MargretFinnboga — APPROVED** [review]
😎

**Pull request merged**